freeipa/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClient.java

```diff
@@ -51,11 +51,13 @@ public FreeIpaClient(JsonRpcHttpClient jsonRpcHttpClient, String apiVersion) {
         return invoke("user_find", flags, params, type);
     }
 
-    public RPCResponse<User> userAdd(String user, String firstName, String lastName) throws FreeIpaClientException {
+    public RPCResponse<User> userAdd(String user, String firstName, String lastName, String password) throws FreeIpaClientException {
         List<String> flags = List.of(user);
         Map<String, Object> params = Map.of(
                 "givenname", firstName,
-                "sn", lastName
+                "sn", lastName,
+                "userpassword", password,
+                "setattr", "krbPasswordExpiration=20380101000000Z"
```

Comment:
Approving this and filed separate JIRA: https://jira.cloudera.com/browse/DISTX-144 for moving krbPasswordExpiration value in config.